tests/drives_tests.cpp

```diff
@@ -88,6 +98,19 @@ TEST(WildFileCmp, QuestionMark)
     EXPECT_EQ(true, WildFileCmp("TEST", "???T.???"));
 }
 
+TEST(WildFileCmp, LongCompare)
+{
+    EXPECT_EQ(false, WildFileCmp("TEST", "", true));
+    EXPECT_EQ(true, WildFileCmp("TEST.EXE", "*", true));
+    EXPECT_EQ(true, WildFileCmp("TEST", "?EST", true));
+    EXPECT_EQ(false, WildFileCmp("TEST", "???Z", true));
+    EXPECT_EQ(true, WildFileCmp("TEST.EXE", "T*T.*", true));
```

Comment:
These are LFN unit tests, but the filenames are all 8.3; can you add some LFN tests, including some that go beyond the max length to test the error handling (or whatever we expect beyond limits?)

Reply:
Good suggestion! I am now adding more unit tests including very long names.